Create a graphical ui representing for an tezos store (#2270)
mirage/irmin: src/irmin-pack-tools/README.md

## tezos-explorer
TODO

## tezos-explorer-gui

> This tool is a graphical UI, meant to allow the user to figure out rapidly the shape of a commit, giving him informations on it's content.

Reviewer: ```suggestion
This tool is a graphical UI, meant to allow the user to figure out rapidly the shape of a commit, giving them information on its content.
```

> In order to launch it, uncomment the `dune` file under the path `src/irmin-pack-tools/tezos_explorer_gui` and install the deps `tsdl` and `tsdl-ttf`.

Reviewer: Add a note about the prettree pin dependency as well.

It can be launched using the following command:
```shell
$ dune exec -- irmin-tezos-explorer-gui <path-to-store> <path-to-ttf-font> <commit>
```

The first argument is the path to the root of the store (e.g. `output/root/`).

The second argument is the path to a `.ttf` file, necessary to know which font to use when printing strings.

> The third argument is an int, the `nth` commit stored in the index of the store that will be showned first.

Reviewer: ```suggestion
The third argument is an int, the `nth` commit stored in the index of the store that will be shown first.
```
